### ISSUE TITLE
Move core-js into dev-dependency

### ISSUE THREAD

**Issue by codysherman** [codysherman/2nfm#215]
Confirm whether `core-js` should be a dev dependecy or not, and if so, move it into the proper location and re-run `yarn`.

**Comment by samtheprogram** [comment]
I don't think this matters, since the project is built via Webpack (by the Vue CLI) before our code can be run anyway; `core-js` is loaded when the project is compiled, which implies use of other `devDependencies`, right? Given the pattern followed with other dependencies, I think it makes sense in `devDependencies` since it's used for polyfilling (just "compile-time overhead" to this project) and not imported directly in any source (to my knowledge).
